queue_services/entity-emailer/tests/unit/test_worker.py

```diff
@@ -202,17 +202,22 @@ def test_process_bn_email(app, session):
             assert mock_send_email.call_args[0][0]['content']['attachments'] == []
 
 
-@pytest.mark.parametrize(['option', 'nr_number', 'subject', 'expiration_date', 'refund_value'], [
-    ('before-expiry', 'NR 1234567', 'Expiring Soon', None, None),
-    ('expired', 'NR 1234567', 'Expired', None, None),
-    ('renewal', 'NR 1234567', 'Confirmation of Renewal', '2021-07-20T00:00:00+00:00', None),
-    ('upgrade', 'NR 1234567', 'Confirmation of Upgrade', None, None),
-    ('refund', 'NR 1234567', 'Refund request confirmation', None, '123.45')
+names_arr = [{'name': 'TEST Company Name', 'state': 'APPROVED'}]
+
+
+@pytest.mark.parametrize(['option', 'nr_number', 'subject', 'expiration_date', 'refund_value', 'names'], [
+    ('before-expiry', 'NR 1234567', 'Expiring Soon', None, None,
```

Comment:
can we add a row that is essentially the same as line 209 that tests the `CONDITION` name state works?

Reply:
done and pushed